GitUI/UserControls/RevisionGrid/Graph/Lane.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace GitUI.UserControls.RevisionGrid.Graph
+{
+    public sealed class Lane
```

Comment:
Does it need to be a class? Can it be a struct? Readonly struct?

Reply:
Indeed, it should be a record.
Shall this one-liner stay in its own file? Or can / should it be moved to `IRevisionGraphRow.cs`?